components/datetime/src/any/datetime.rs

```diff
@@ -15,11 +15,11 @@ use icu_provider::prelude::*;
 use crate::provider::{
     calendar::{
         DatePatternsV1Marker, DateSkeletonPatternsV1Marker, DateSymbolsV1Marker,
-        TimePatternsV1Marker, TimeSymbolsV1Marker,
+        TimeLengthsV1Marker, TimeSymbolsV1Marker,
```

Comment:
Nit: We have DatePatternsV1Marker mapping to datelengths@1, too. We should be consistent in the naming, by either keeping the status quo, changing the keys to say "patterns", or changing the structs to say "lengths"